app/views/ProjectEdit/Framework/index.tsx

```diff
@@ -156,37 +172,36 @@ function ProjectFramework(props: Props) {
     } = useForm(schema, defaultFormValue);
 
     const delayedValue = useDebouncedValue(value);
-
     useEffect(() => {
         setOffset(0);
         setFrameworkList([]);
     }, [delayedValue]);
```

Comment:
While you are here, this should also be removed. Can you make sure to update frameworkList and offset as soon as `value` is changed?